pytest_axe/parametrize_tests.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+import pytest
+from axe_selenium_python import Axe
+from selenium import webdriver
+
+
+def format_parameter(rule, xfail_rules):
+    return pytest.param(rule, marks=pytest.mark.xfail(reason=xfail_rules[rule]))
+
+
+def axe_core_get_rules():
+    os.environ["MOZ_HEADLESS"] = "1"
+    driver = webdriver.Firefox()
+    axe = Axe(driver)
+    axe.inject()
+    rules = driver.execute_script("return axe.getRules()")
+    driver.close()
+    return rules
+
+
+def parametrized_accessibility_rules(metafunc):
+    rules = axe_core_get_rules()
+
```

Comment:
A nit which you can ignore or act on, your choice. Add some comments to highlight what's happening in this method.

Reply:
Good point. I get so into trying to make it work I often forget to document!